native-stack/README.md

```diff
@@ -287,6 +287,7 @@ How the given screen should appear/disappear when pushed or popped at the top of
 - `slide_from_bottom` – performs a slide from bottom animation
 - `slide_from_right` - slide in the new screen from right to left (Android only, resolves to default transition on iOS)
 - `slide_from_left` - slide in the new screen from left to right (Android only, resolves to default transition on iOS)
+- `cupetino` - iOS like slide in animation (Android only, resolves to default transition on iOS)
```

Comment:
Same as above

Reply:
Just want to note I'm expert at making rypos :D